epicure-frontend/src/components/Footer/Footer.tsx

```diff
@@ -0,0 +1,17 @@
+import "./Footer.scss";
+
+const Footer = () => {
+  return (
+    <>
+      <footer className='footer'>
+        <div className='footer-links'>
+          <div>Contact Us</div>
+          <div>Term of Use</div>
+          <div>Privacy Policy</div>
```

Comment:
use links and not divs set the path to / for 
now and make it as const of list with map